components/data-table/column.jsx

```diff
@@ -41,47 +41,14 @@ DataTableColumn.propTypes = {
 	 * </DataTable>
 	 * ```
 	 */
-	children: PropTypes.element,
 	/**
 	 * Some columns, such as "date last viewed" or "date recently updated," should sort descending first, since that is what the user probably wants. How often does one want to see their oldest files first in a table? If sortable and the `DataTable`'s parent has not defined the sort order, then ascending (A at the top to Z at the bottom) is the default sort order on first click.
 	 */
 	isDefaultSortDescending: PropTypes.bool,
-	/**
-	 * Selects this column as the currently sorted column.
-	 */
-	isSorted: PropTypes.bool,
-	/**
-	 * The column label. If a `string` is not passed in, no `title` attribute will be rendered.
-	 */
-	label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
Same here.

Reply:
Thanks for identifying it. I lokked it and some propTypes are used which I have removed. I will correct it.